zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/compensation/CompensationEventExecutionTest.java

```diff
@@ -2230,6 +2231,268 @@ public void shouldCompensateSubprocessWithInnerCompensationHandler() {
             tuple(PROCESS_ID, ProcessInstanceIntent.ELEMENT_COMPLETED));
   }
 
+  @Test
+  public void shouldInvokeCallActivityCompensationHandler() {
```

Comment:
❓ This test case and `shouldCompensateCallActivity` can be merged together, what do you think ?

Reply:
Yes, it would be possible to merge both test cases. However, I see value in testing the compensation activity and the compensation handler in separate test cases, following the idea of "one assert per test".